kitsune/search/v2/base.py

```diff
@@ -86,8 +86,7 @@ def to_action(self, action=None, **kwargs):
         payload = {}
         # if we have a bulk update, we need to include the meta info
         # and return the data by calling the to_dict() method of DSL
-        if is_bulk := kwargs.pop("is_bulk", False):
-            payload = self.to_dict(include_meta=is_bulk)
+        payload = self.to_dict(include_meta=is_bulk, skip_empty=False)
```

Comment:
Why do we need to keep empty values?

Reply:
Otherwise these tests will fail: https://github.com/mozilla/kitsune/pull/4602/files#diff-f270bdc2c6b7be81ab0afaa0979895167bb9506dbfd5bbacde9b3f9fda3be529R34-R41

A concrete example would be if a kb article was removed from all topics, without `skip_empty=False` when it was updated it would remain in its previous set of topics in ES.